core/src/main/kotlin/com/alecstrong/sql/psi/core/SqlFileBase.kt

```diff
@@ -167,18 +165,18 @@ abstract class SqlFileBase(
   }
 
   private inline fun iteratePreviousStatements(block: (SqlStmt) -> Unit) {
-    val files = SparseArrayCompat<SqlFileBase>()
+    val files = mutableMapOf<Int, SqlFileBase>()
```

Comment:
`toSortedMap` reminds me - is there a `sortedMapOf` which would then keep it sorted on inserts?

(i'm extra careful for performance in these parts of the code since they run essentially on every key stroke)

Reply:
Fixed in https://github.com/AlecStrong/sql-psi/pull/148/commits/c1079508fe44a4613e58a989246ad0f3584ffeba.